DEPENDENCIES.rst

```diff
@@ -25,16 +25,17 @@ dependencies is shown below:
 ====================   ==================
 Package                Minimum Version
 ====================   ==================
-``CMake``                2.8
-``boost``                1.46.1
-``libxml2``              2
-``libxml++``             2.36
-``python``               2.7 or 3.3+
-``sqlite3``              3.7.10
-``HDF5``                 1.8.4
-``Coin-Cbc``             2.5
+``CMake``                3.16.3
+``boost``                1.71.0
+``libxml2``              2.9.10+
+``libxml++``             2.40.1
+``python``               3.8.10
+``sqlite3``              3.31.1
+``HDF5``                 1.10.4+
+``Coin-Cbc``             2.10.3
+``Coin-Clp``             1.17.5
+``zlib``                 1.2.11
 ====================   ==================
-
 .. website_include_end
 
 On some platforms, such as Ubuntu 16.04, the following are also necessary:
```

Comment:
Do we want to keep this?  I'd say we leave 16.04 behind.  Are there other special platforms to call out?

Reply:
We could note that `libxml++` needs to be built from source in Rocky builds?

Reply:
Does it though? On my machine (running Rocky 9.0) I have libxml++ and libxml++-4.0 from conda. Also, are enough people running on Rocky machines to warrant calling it out?

Reply:
True... libxml++ only needs to be built from source if you don't use conda on Rocky.  I'll leave it out for now.  If we receive feedback about difficulties building on Rocky we can re-evaluate.